pkg/operator/starter.go

```diff
@@ -40,6 +46,94 @@ const (
 	resyncInterval = 20 * time.Minute
 )
 
+
+func addDCObjectHash(deployment *appsv1.Deployment, inputHashes map[string]string) error {
```

Comment:
Should we start using generics so that we don't have to duplicate the function for a different type?

Reply:
We _could_, but tbh I'd like this to go away asap on `master` (as soon as the change to `library-go` merges) so it doesn't seem worth it.